pandas_gbq/gbq.py

```diff
@@ -495,6 +496,11 @@ def run_query(self, query, max_results=None, progress_bar_type=None, **kwargs):
         except self.http_error as ex:
             self.process_http_error(ex)
 
+        # Avoid attempting to download results from DML queries, which have no
+        # destination.
```

Comment:
This will also impact scripts as well; do you need additional handling for that case (example: the last line in a script is a SELECT)

Reply:
That's on the TODO list: https://github.com/googleapis/python-bigquery-pandas/issues/149#issuecomment-451559468

The `to_dataframe` method in `google-cloud-bigquery can call `getQueryResults` now, which should handle the scripting case. Though I'm actually not sure what happens when we try to use the BQ Storage API in `google-cloud-bigquery` with a script. Filed https://github.com/googleapis/python-bigquery/issues/1148 to investigate this in `google-cloud-bigquery`.